react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -603,6 +603,15 @@ class RCTAztecView: Aztec.TextView {
         let caretData = packCaretDataForRN()
         onSelectionChange(caretData)
     }
+
+    // MARK: - Selection
+    private func correctSelectionAfterZWSP() {
+        guard selectedTextRange?.start == endOfDocument,
+            text == String(.zeroWidthSpace) else {
```

Comment:
Do you think we could associate the check with this configuration in common init: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1552/files#diff-ad68d1bf3bf68556db4be6e6215573adL136

Just in case the character changes in the future?

Reply:
Good catch, pushed a change to make it more generic